barrantes/book_journal.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Book Details</title>
+</head>
+<body>
+    <h1>Book Details</h1>
+    <h2>Create Book Entry</h2>
+    <form id="createForm">
+        <label for="Book_Name">Book Name:</label>
+        <input type="text" id="Book_Name" name="Book_Name" required><br><br>
+        
+        <label for="Book_Genre">Book Genre:</label>
+        <input type="text" id="Book_Genre" name="Book_Genre" required><br><br>
+        
+        <label for="Release_Date">Release Date:</label>
+        <input type="date" id="Release_Date" name="Release_Date" required><br><br>
+        
+        <label for="Author">Author:</label>
+        <input type="text" id="Author" name="Author" required><br><br>
+        
+        <label for="Rating">Rating:</label>
+        <input type="number" id="Rating" name="Rating" required><br><br>
+        
+        <button type="submit" onclick="submitUser()">Save Book</button>
+    </form>
+    <h2>Update Book Entry</h2>
+    <form id="updateForm">
+        <label for="Update_ID">ID:</label>
+        <input type="text" id="Update_ID" name="Update_ID" required><br><br>
+
+        <!-- Fields for updating existing book entry -->
+        <label for="Update_Book_Name">Updated Book Name:</label>
+        <input type="text" id="Update_Book_Name" name="Update_Book_Name"><br><br>
+        
+        <label for="Update_Book_Genre">Updated Book Genre:</label>
+        <input type="text" id="Update_Book_Genre" name="Update_Book_Genre"><br><br>
+        
+        <label for="Update_Release_Date">Updated Release Date:</label>
+        <input type="date" id="Update_Release_Date" name="Update_Release_Date"><br><br>
+        
+        <label for="Update_Author">Updated Author:</label>
+        <input type="text" id="Update_Author" name="Update_Author"><br><br>
+        
+        <label for="Update_Rating">Updated Rating:</label>
+        <input type="number" id="Update_Rating" name="Update_Rating"><br><br>
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)